int32 box_number

--------------------

bool acknowledged